lisa/tools/lspci.py

```diff
@@ -83,12 +85,10 @@ def get_device_list(self, force_run: bool = False) -> List[PciDevice]:
             self._pci_devices = []
             result = self.run("-m", force_run=force_run, shell=True)
             if result.exit_code != 0:
-                result = self.run("-m", force_run=force_run, shell=True, sudo=True)
-                if result.exit_code != 0:
-                    raise LisaException(
-                        f"get unexpected non-zero exit code {result.exit_code} "
-                        f"when run {self.command} -m."
-                    )
+                raise LisaException(
```

Comment:
The simple exit_code check can be merged into `self.run` by using `expected_exit_code` and `expected_exit_code_failure_message`.

Reply:
fixed